src/thunderbots/software/test/evaluation/player.cpp

```diff
@@ -0,0 +1,80 @@
+/**
+ * This file contains the unit tests for evaluation function
+ * playerOrientationWithinAngleThresholdOfTarget in player.cpp
```

Comment:
nit, but we should probably change this to something like:
```
 * This file contains the unit tests for evaluation functions
 * in player.cpp
```
so that it doesn't go out-of-date when we add more evaluation functions